sound/squeezelite/Makefile

```diff
@@ -6,13 +6,13 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=squeezelite
-PKG_VERSION:=1.9.9-1415
+PKG_VERSION:=1.9.9-1432
```

Comment:
This update should be done in separate commit.

Reply:
OK, I created #21096